src/content/docs/release-notes/mobile-release-notes/react-native-release-notes/react-native-agent-146.mdx

```diff
@@ -1,7 +1,7 @@
 ---
 subject: React Native agent
 releaseDate: '2024-09-16'
-version: 1.4.5
+version: 1.4.6
 downloadLink: 'https://www.npmjs.com/package/newrelic-react-native-agent/v/1.4.5'
```

Comment:
Please check the link for v1.4.5
It should be:
downloadLink: 'https://www.npmjs.com/package/newrelic-react-native-agent/v/1.4.6'

Reply:
looks good